bpf-rust-programs/prediction-poll/program/do.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")"
+
+usage() {
+    cat <<EOF
+
+Usage: do.sh action <project>
+
+If relative_project_path is ommitted then action will
+be performed on all projects
+
+Supported actions:
+    build
+    clean
+    test
+    clippy
+    fmt
+
+EOF
+}
+
+perform_action() {
+    set -e
+    case "$1" in
+    build)
+         ../../../node_modules/@solana/web3.js/bpf-sdk/rust/build.sh "$PWD"
+         mkdir -p ../../../dist/programs
+         cp ../target/bpfel-unknown-unknown/release/prediction_poll.so ../../../dist/programs
+        ;;
+    clean)
+         ../../../node_modules/@solana/web3.js/bpf-sdk/rust/clean.sh "$PWD"
+        ;;
+    test)
+            echo "test $2"
```

Comment:
Nit, `$2` should be removed.  I haven't cleaned that up in the other repos yet but might as well start here.